File: M03/UF1/Pt2/ej1.java
import java.util.Scanner;
class pt2_ej1 {
    public static void main(String[] args) throws InterruptedException {
        Scanner sc = new Scanner(System.in);
        System.out.print("Inserta un numero -->  ");
        int num = sc.nextInt();
        String par_o_senar = "";
        if (num % 2 == 0) {
            par_o_senar = "PARELL";
        }
        else {
            par_o_senar = "SENAR";
        }
        Thread.sleep(500);
        System.out.println("El numero introduciso es " + par_o_senar);
        sc.close(); 
    }
}

File: M03/UF1/Pt2/ej1.py
#Feu un programa que llegeixi un número sencer, i ens digui si és parell o senar.
import time

num = float(input("Introduce un numero -->  "))
par_o_senar = "PARELL" if (num % 2 == 0) else "SENAR"

time.sleep(0.5)

print(f"El numero introducido es {par_o_senar}")